src/actions/document.js

```diff
@@ -1,5 +1,17 @@
 export const UPDATE_MARKDOWN = 'UPDATE_MARKDOWN';
 export const updateMarkdown = markdown => ({
   type: UPDATE_MARKDOWN,
-  payload: { markdown }
+  payload: markdown
+});
+
+export const UPDATE_FILES = 'UPDATE_FILES';
+export const updateFiles = file => ({
```

Comment:
the card asks to display markdown files. this seems to be about this card: https://trello.com/c/wt5ED67E/8-create-container-to-save-markdown-files-4pts